mq/esworker_papertrail.py

```diff
@@ -547,4 +547,4 @@ def initConfig():
     lastPluginCheck = datetime.now()-timedelta(minutes=60)
     pluginList, lastPluginCheck = checkPlugins(pluginList, lastPluginCheck)
 
-    main()
+main()
```

Comment:
We lost the indentation here, which needs to be there since we want to call 'main' only if we're running this python script.

Reply:
ok, will fix